### PR TITLE
refactor(nprogress): Upgrade to latest remix version with vite

### PR DESCRIPTION
**Pull request by machour**

### CODE REVIEW

**Comment by machour** [comment]
CI seems to have a little problem

**Review by sergiodxa — APPROVED** [review]
I think entry.client and entry.server files could be removed? They are the default ones.

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16
+18
```
Comment (sergiodxa) [inline .nvmrc:1]: What if this is set to `lts/*`? Remix should always work with current LTS, right?

Comment (machour) [reply .nvmrc:1]: Not sure what's the team policy here. Any thoughts @brookslybrand ?

**Comment by machour** [comment]
@brookslybrand You were right, it's the `.eslintrc.cjs` file presence that makes the CI fail.
It's the same one that gets installed by the template, ie, https://github.com/remix-run/remix/blob/main/templates/remix/.eslintrc.cjs

**Comment by brookslybrand** [comment]
> @brookslybrand You were right, it's the `.eslintrc.cjs` file presence that makes the CI fail. It's the same one that gets installed by the template, ie, https://github.com/remix-run/remix/blob/main/templates/remix/.eslintrc.cjs

Hmm, yeah, seems like something not playing well with the root eslint which is run in CI/CD. Running it locally it _seems_ like the linting you added passes

**Pull request merged**